ui/src/components/pages/Pipeline/partials/Graph/partials/NodeInfo/partials/Pods/partials/PodDetails/partials/Metrics/partials/LineChart/index.tsx

```diff
@@ -352,7 +361,7 @@ const LineChartComponent = ({
         </Box>
       )}
 
-      {!isLoading && error && <EmptyChart />}
+      {!isLoading && error && <EmptyChart message={error.toString() || "No data for the selected filters."}/>}
```

Comment:
we do not need to pass `"No data..."`, as it is handled in case of `undefined`. also lets use optional chaining like above.